Resources/Prototypes/Entities/Objects/Fun/Instruments/instruments_string.yml

```diff
@@ -8,8 +8,8 @@
     program: 27
   - type: SwappableInstrument
     instrumentList:
+      "Jazz": {26: 0}
```

Comment:
Since the default program is 27, i.e. Clean, it means that Clean should come first in the `instrumentList` to ensure that it is the first option in the rightclick menu. 

Either swap the default program to 26 (making Jazz the default), or move the "Clean: {27: 0}" line to the top again.

Reply:
As wacky as it looks in code, swapping it so that Clean is the default feels right.